### PR TITLE
DCM-48: Add save/update functionality to location mapping UI

### PR DESCRIPTION
**Pull request by Piumal1999**
## The issue I worked on
See [https://issues.openmrs.org/browse/DCM-48](https://issues.openmrs.org/browse/DCM-48) 

## Description of what I changed:
Add Saving/Updating feature to the location mapping page

image

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java [hunk 1]
```diff
@@ -1644,4 +1644,14 @@ public String runAllAutomatedReportsAndPostToDHIS() {
 	public List<LocationToOrgUnitMapping> getAllLocationToOrgUnitMappings() {
 		return getDao().getAllLocationToOrgUnitMappings();
 	}
+
+	@Override
+	public void saveLocationToOrgUnitMapping(LocationToOrgUnitMapping locationToOrgUnitMapping) {
```
Comment (HerbertYiga) [inline api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java:1649]: thanks @Piumal1999  could you create tests for the added methods

Comment (Piumal1999) [reply api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java:1649]: @HerbertYiga Ok sure. I'll update the pull request.

Comment (Piumal1999) [reply api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java:1649]: Added the tests with https://github.com/openmrs/openmrs-module-dhisconnector/commit/16e8fd2c29edb38bcfbae751bf338cddf1735eb5

**Comment by akshika47** [comment]
@jayasanka-sack would you be able to take look at this PR?

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Piumal1999 !

**Pull request merged**